src/components/Inspector/InspectorCard.module.css

```
@@ -82,29 +82,40 @@
   color: var(--dc-fg-muted);
 }
 
-/* Mobile tier: tighten card padding so the controls keep breathing room
-   against the rest of the inspector. */
-:global(.app-container[data-layout-tier="mobile"]) .cardHead {
+/* Mobile tier + sheet placement: tighten card padding so the controls keep
+   breathing room against the rest of the inspector. The sheet hook covers
+   tablet-split, where the vaul portal re-stamps tier="tablet" and the
+   mobile-tier guard alone wouldn't reach. */
+:global([data-layout-tier="mobile"]) .cardHead,
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)